staging/src/k8s.io/kubectl/pkg/cmd/config/unset.go

```diff
@@ -33,25 +35,31 @@ import (
 type unsetOptions struct {
 	configAccess clientcmd.ConfigAccess
 	propertyName string
+	jsonPath     bool
+
+	streams genericclioptions.IOStreams
 }
 
 var (
 	unsetLong = templates.LongDesc(i18n.T(`
 	Unset an individual value in a kubeconfig file.
 
-	PROPERTY_NAME is a dot delimited name where each token represents either an attribute name or a map key.  Map keys may not contain dots.`))
+	PROPERTY_NAME is either a jsonpath query for where the property name will be, or a dot delimited name.`))
```

Comment:
```suggestion
	PROPERTY_NAME is either a jsonpath query for where the property name will be, or a dot delimited name.`))
```
```suggestion
	PROPERTY_NAME is either a dot-eliminated field path, or a jsonpath query which identifies the properties to act upon`))
```

Reply:
This may be a bit pedantic, but I think that "identifies the properties to act upon" is inaccurate because it is possible that the property the user is trying to set does not exist yet, and so when I say "where the properties will be" I specifically mean that the path is where it will be if it does not already exist, or where it will be found if it does exist. Agree that I should not say "property name" and just "properties" is better.